p2panda-tests/src/utils.rs

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
Multiple files miss the SPDX license headers, we should maybe add a checkbox as a reminder into the PR template?

Reply:
doooone